### PR TITLE
Codacy Integration

### PR DESCRIPTION
**Pull request by sidhyatikku**
Integrated Codacy & Added the badge to `readme.md` #1768 

---
This change is [Reviewable](https://reviewable.io/reviews/moneymanagerex/moneymanagerex/1887)

### CODE REVIEW

**Comment by slodki** [comment]
The link is not working (?).

File: .codacy.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+engines:
+ rubocop:
+   enabled: true
+   exclude_paths:
+     - config/engines.yml
```
Comment (slodki) [inline .codacy.yml]: we don't have such file in this repo

File: README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ Money Manager Ex
 [![stable]][GitHubLatest] [![unstable]][GitHubDL] [![s]][SourceForgeDL]
 [![gha]][GitHubDL] [![ghl]][GitHubLatest] [![p]][packagecloud] [![w]][website]
 [![i]][GitHubIssues] [![resolution time]][isitmaintained] [![cd]][CodeDocs]
-[![c]][contributors] [![l]](LICENSE) [![o]][OpenHub]
+[![c]][contributors] [![l]](LICENSE) [![o]][OpenHub] [![Codacy Badge](https://api.codacy.com/project/badge/Grade/4cb8bd40c2304ae88251087be6b243c9)](https://www.codacy.com/app/=moneymanagerex/moneymanagerex?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=moneymanagerex/moneymanagerex&amp;utm_campaign=Badge_Grade)
```
Comment (slodki) [inline README.md]: Link not working
Are utm params required?
Please move URLs to the end of the file.

File: .codacy.yml (hunk 1 above)
Comment (sidhyatikku) [reply .codacy.yml]: My bad, I just copy pasted from my repo 😅

File: README.md (hunk 2 above)
Comment (slodki) [reply README.md]: We shouldn't put links to private resources in the README.

How to verify the `4cb8bd40c2304ae88251087be6b243c9` project is really mmex?

**Comment by slodki** [comment]
https://support.codacy.com/hc/en-us/articles/360006718454-Share-your-project-with-a-non-Codacy-user should be useful here

**Comment by sidhyatikku** [comment]
@slodki This will only give access to the dashboard of my fork of moneymanagerx. I would suggest you to the app via logging into your codacy account and request a badge via codacy badger for this repo.

**Comment by GitCop** [comment]
Thanks for your pull request while there were the following issues

 * Commit: dbeaa8aaf55cb590ec4f02117c791c46629ceecb
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 885b194020986a089b817151b0d92bf0831776bb
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: a15f28a5866e8562bc772edce96a0895fae6b30b
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2b929aa1e2d453763f46b4378660bba6b2b4455b
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/moneymanagerex/moneymanagerex/wiki/GitCop

---
This message was auto-generated by https://gitcop.com

**Comment by slodki** [comment]
@sidhyatikku I can see only my fork @codacy only. I think only @guanlisheng as GH maintainer can follow https://support.codacy.com/hc/en-us/articles/212622529-Why-can-t-I-see-my-organization-s-projects- instructions...

BTW: I cannot see any *Share* button at a Codacy Dashboard page.

**Comment by sidhyatikku** [comment]
@slodki only repo owners can view the dashboard and enable cpdacy on repo. I’ve added .codacy.yml and once codacy is enabled by the repo admin, it’ll start working.

**Comment by sidhyatikku** [comment]
@slodki Just enable codacy on the repo via your dashboard.

**Comment by slodki** [comment]
MMEX should be visible now under https://app.codacy.com/project/slodki/moneymanagerex/dashboard without account creation/login.
slodki in the above URL is codacy username - above link is for moneymanagerex/moneymanagerex (not slodki/moneymanagerex) repository - you can see PR listed.
I wonder how to reallocate MMEX from https://app.codacy.com/project/slodki/moneymanagerex/dashboard to https://app.codacy.com/project/moneymanagerex/moneymanagerex/dashboard address? I thing moneymanagerex GitHub user must create codacy account and enable MMEX repo...

**Comment by sidhyatikku** [comment]
@slodki codacy has been integrated successfully. You may check the `All 3 checks have passed`.

**Comment by slodki** [comment]
#1907

**Pull request closed without merging**